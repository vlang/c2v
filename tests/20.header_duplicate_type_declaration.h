#pragma once

#include "IGNORE_header_duplicate_type_declaration_2.h"

typedef int dup_typ;
