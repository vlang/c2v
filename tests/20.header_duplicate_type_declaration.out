[translated]
module tests

type Dup_typ = int
